src/main/java/racingcar/RaceSnapshot.java

```diff
@@ -0,0 +1,12 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RaceSnapshot {
+    List<CarSnapshot> carList = new ArrayList<CarSnapshot>();
```

Comment:
접근제어자를 잊지 말아주세요! 😢
또한 이전 피드백에서 말씀드렸떤 사항인데요 carList 보다는 cars 가 어떨까요? 😄 
 ```suggestion
    private List<CarSnapshot> cars;
```